### PR TITLE
Update rubygem-foreman_remote_execution to 2.0.0

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by ares** [comment]
[test rpm]

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-Version:    1.8.4
+Version:    2.0.0
 Release:    1%{?foremandist}%{?dist}
 Group:      Applications/System
 License:    GPLv3
 URL:        https://github.com/theforeman/foreman_remote_execution
 Source0:    https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires:   foreman >= 1.20.0
+Requires:   foreman >= 1.24.0
```
Comment (lzap) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: Also `BuildRequires: foreman-plugin >= 1.24.0` is needed I think.

**Comment by lzap** [comment]
I am going to replace this PR with the same one but with foreman-cockpit changes.

**Comment by lzap** [comment]
Here is my PR: https://github.com/theforeman/foreman-packaging/pull/4207

**Comment by adamruzicka** [comment]
Closing in favor of #4207

**Pull request closed without merging**